diff --git a/api/python/provisioner/commands/__init__.py b/api/python/provisioner/commands/__init__.py
index 7cc047c6..868c4ed3 100644
--- a/api/python/provisioner/commands/__init__.py
+++ b/api/python/provisioner/commands/__init__.py
@@ -1048,7 +1048,7 @@ class GetReleaseVersion(CommandParserFillerMixin):
     @property
     def installed_rpms(self) -> List:
         if self._installed_rpms is None:
-            exclude_rpms = 'cortx-py|prvsnr-cli'
+            exclude_rpms = 'cortx-py|prvsnr-cli|cortx-sspl-test'
             res = salt_cmd_run(f"rpm -qa|grep '^cortx-'|grep -Ev '{exclude_rpms}'",  # noqa: E501
                                targets=LOCAL_MINION)
             rpms = res[next(iter(res))].split("\n")
