--- /opt/seagate/eos/hare/libexec/build-ees-ha  2020-03-07 19:13:18.393211960 +0000
+++ /opt/seagate/eos/hare/libexec/build-ees-ha.new      2020-03-07 19:39:24.301279343 +0000
@@ -183,8 +183,8 @@
 # Update data_iface values in CDF: 1st data_iface will be `${iface}_c1`,
 #                                  2nd data_iface will be `${iface}_c2`.
 sudo sed -E -e "/[#].*data_iface/b ; # skip commented out data_iface lines
-                0,/(data_iface: *)[a-z]+[0-9]([^_]|$)/s//\1${iface}_c1/ ;
-                0,/(data_iface: *)[a-z]+[0-9]([^_]|$)/s//\1${iface}_c2/" \
+                0,/(data_iface: *)\b${iface}\b/s//\1${iface}_c1/ ;
+                0,/(data_iface: *)\b${iface}\b/s//\1${iface}_c2/" \
             -i $cdf

 hctl bootstrap --mkfs $cdf
